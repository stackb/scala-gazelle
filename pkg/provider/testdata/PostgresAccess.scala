package auth.dao

object PostgresAccess extends common.postgres.PostgresAccess {
  override def dbConfigKey: String = "auth"
}
