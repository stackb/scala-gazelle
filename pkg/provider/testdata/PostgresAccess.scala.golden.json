{
  "label": "//pkg/provider:testdata/PostgresAccess.scala",
  "kind": "scala_library",
  "files": [
    {
      "filename": "testdata/PostgresAccess.scala",
      "packages": [
        "auth.dao"
      ],
      "objects": [
        "auth.dao.PostgresAccess"
      ],
      "names": [
        "PostgresAccess",
        "String",
        "auth.dao",
        "common.postgres.PostgresAccess",
        "dbConfigKey"
      ],
      "extends": {
        "object auth.dao.PostgresAccess": {
          "classes": [
            "common.postgres.PostgresAccess"
          ]
        }
      }
    }
  ]
}